<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>738</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 129, 183);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ПРИВЕТ!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>КАК СО МНОЙ РАБОТАТЬ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> 1. НАЖМИТЕ НА ПОЛЕ ВВОДА И ВЫБЕРИТЕ ФАЙЛ, </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>534</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2. НАЖМИТЕ НА &quot;ОК&quot;, ЧТОБЫ ДОБАВИТЬ ФАЙЛ, </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>534</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3. ЕСЛИ ХОТИТЕ ДОБАВИТЬ ЕЩЕ ФАЙЛЫ, </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>534</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>4. ДАЛЕЕ ВЫБЕРИТЕ ТИП АНАЛИЗА ДАННЫХ.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>534</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>5. НАЖМИТЕ НА КНОПКУ &quot;ГОТОВО!&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>КОТОРЫЙ БУДЕТЕ ИССЛЕДОВАТЬ.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>И  - НА &quot;CANCEL&quot;, ЧТОБЫ ЕГО НЕ ДОБАВЛЯТЬ.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ТО ПОВТОРИТЕ ПУНКТЫ 1 И 2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(5, 51, 110);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>main_logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ChosenFilesLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(5, 51, 110);</string>
    </property>
    <property name="text">
     <string>Выбранные Файлы:   (0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(5, 51, 110);</string>
    </property>
    <property name="text">
     <string>ВЫБЕРИ ТИП АНАЛИЗА:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AnalysisType1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>type_1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(111, 138, 188);
background-color: rgb(5, 51, 110);</string>
    </property>
    <property name="text">
     <string>Analysis (conventional)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AnalysisType2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>289</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>type_2</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(147, 115, 168);
background-color: rgb(5, 51, 110);</string>
    </property>
    <property name="text">
     <string>Analysis (model based)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AnalysisType3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>620</y>
      <width>289</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>type_3</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(129, 177, 101);
background-color: rgb(5, 51, 110);</string>
    </property>
    <property name="text">
     <string>Analysis (complicated)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonConfirm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>660</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 136, 12);</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="text">
     <string>ГОТОВО!</string>
    </property>
   </widget>
   <widget class="Line" name="border">
    <property name="geometry">
     <rect>
      <x>467</x>
      <y>0</y>
      <width>20</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 49, 83);
color: rgb(23, 49, 83);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="left_side_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(5, 51, 110);
border-color: rgb(5, 51, 110);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileSearchLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 150);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Добавь нужные файлы: </string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="AccpeptFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить файл</string>
    </property>
   </widget>
   <widget class="QListWidget" name="FilesList">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <zorder>left_side_bar</zorder>
   <zorder>label_6</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_5</zorder>
   <zorder>label_7</zorder>
   <zorder>label_4</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_9</zorder>
   <zorder>ChosenFilesLabel</zorder>
   <zorder>label_3</zorder>
   <zorder>AnalysisType1</zorder>
   <zorder>AnalysisType2</zorder>
   <zorder>AnalysisType3</zorder>
   <zorder>border</zorder>
   <zorder>logo</zorder>
   <zorder>FileSearchLineEdit</zorder>
   <zorder>AccpeptFile</zorder>
   <zorder>PushButtonConfirm</zorder>
   <zorder>AddFile</zorder>
   <zorder>FilesList</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
